src/main/java/org/opensrp/repository/LocationRepository.java

```diff
@@ -274,4 +276,22 @@
 	 * @return number of records
 	 */
 	long countLocationsByIds(List<String> locationIds, long serverVersion);
+
+	/**
+	 * Gets the structures/locations with the stocks attached to them
```

Comment:
Does this method support getting both structures and locations?

Reply:
Structures are locations right ... they are just not jurisdictions.